imap-codec/Cargo.toml

```diff
@@ -19,6 +19,8 @@ default = ["quirk_rectify_numbers", "quirk_missing_text", "quirk_trailing_space"
 fuzz = []
 
 # <Forward to imap-types>
+serde = ["dep:serde", "imap-types/serde"]
```

Comment:
Do the CI test this feature?

Reply:
It does through `--all-features`. That's why I added a tests for the additional derive, as otherwise the additional implementation would have reduced the coverage.